packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -44,39 +44,40 @@ const FeedEmptyScreen = dynamic(
   () => import(/* webpackChunkName: "feedEmpty" */ './FeedEmptyScreen'),
 );
 
+const FeedFilters = dynamic(() => import('./filters/FeedFilters'));
```

Comment:
We could add the chunk name if it's used anywhere else

Reply:
Not used anywhere else as of the moment, but I agree, I think it is a good practice to set the chunk name to be future-proof.

Reply:
Looks like I have pushed it in a different PR. Though that PR is pointing here, I think we can mark this out. 

https://github.com/dailydotdev/apps/pull/1318/files#diff-dc8e96a8b7f7dbbeb512be9e39b798856c9749a3506621c9f1927f79091bc903R47-R50